### PR TITLE
Don't run build and e2e on fix branches

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

File: circle.yml [hunk 1]
```diff
@@ -10,32 +10,10 @@ deployment:
...
-      - testVer=$(cat version.json |tr -d '"') && gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/$testVer/electron*.json
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/*.sh
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/*.exe
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/electron*.json
-      - ./run-e2e-tests.sh
+      - echo "not building / deploying feature/fix/chore branch"
 
   staging:
-    branch: /stage\/.*/
+    branch: /release\/.*/
```
Comment (ahmedalsudani) [inline circle.yml]: @tejohnso is this a regex that matches any branch with "release" in its name?

File: circle.yml [hunk 2]
```diff
@@ -88,15 +66,6 @@ deployment:
...
-      - testVer=$(cat version.json |tr -d '"') && gsutil cp builds/*.exe gs://install-versions.risevision.com/$testVer/
-      - testVer=$(cat version.json |tr -d '"') && gsutil cp cfg/*.json gs://install-versions.risevision.com/$testVer/
-      - testVer=$(cat version.json |tr -d '"') && gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/$testVer/*.sh
-      - testVer=$(cat version.json |tr -d '"') && gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/$testVer/*.exe
-      - testVer=$(cat version.json |tr -d '"') && gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/$testVer/electron*.json
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/*.sh
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/*.exe
-      - testVer=$(cat version.json |tr -d '"') && gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/$testVer/electron*.json
```
Comment (ahmedalsudani) [inline circle.yml]: @tejohnso this is necessary for our E2E runs. I guess we can copy from one bucket to another,  but we need those files in the version directories

File: circle.yml (hunk 1 above)
Comment (tejohnso) [inline circle.yml]: @ahmedalsudani release with forward slash similar to chore/whatever.  So we'll use for example "release/12".

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**